src/react/ui-elements/ListLayout3.jsx

```diff
@@ -0,0 +1,67 @@
+import styled, { css } from 'styled-components';
+
+export const Container = styled.div`
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    background: ${props => props.theme.brand.background};
+    flex: 1;
+`;
+
+export const BreadcrumbContainer = styled.div`
+    margin: 0px 10px;
+    height: 22px;
+    display: flex;
+    min-height: 22px;
+    padding: 4px 0px;
+
+    .sc-breadcrumb_item {
+        font-size: 15px;
+    }
+`;
+
+export const Body = styled.div`
+    display: flex;
+    flex: 1;
+    flex-direction: row;
+    width: 100%;
+
+    margin: 10px 0px;
+`;
+
+export const ListSection = styled.div`
+    display: flex;
+    flex: 1 1 60%;
```

Comment:
According to the design
```suggestion
    flex: 1 1 50%;
```